UI/Panels/Settings/MobiFlightPanel.cs

```diff
@@ -1081,10 +1067,12 @@ private void unregisterMuxClient()
         {
             MobiFlight.Config.MultiplexerDriver multiplexerDriver;
             
-            multiplexerDriver = getModuleMultiplexerDriver();
-            multiplexerDriver.unregisterClient();
-            if (!multiplexerDriver.isInitialized()) {
-                moduleMultiplexerDrivers.Remove(getModuleNode().Name);
+            multiplexerDriver = findModuleMultiplexerDriver(false);
```

Comment:
In general I would propose to rethink about the approach to handle a lot of this MultiplexerDriver logic in the View component.

I would like to handle the unregisterClient Logic inside the Config object.

It seems to me that there is too much knowledge of the Driver required here.

Reply:
Could you elaborate? If I'm not mistaken, the saved/uploaded configuration is sourced from the View component before the corresponding Config object exists; and this latter is rebuilt by reading back the board config. Therefore the driver is created (or deleted) first in the GUI.
In that place, it is required that each new client un/registers itself for the reasons mentioned above; I can't see any useless operation there.
Could you make an example of what the `getModuleMultiplexerDriver()` should do differently (or how it could possibly be removed)?

Reply:
let's schedule an interactive coding session and try some stuff. it costs a lot of time to discuss this here

Reply:
Agreed, I'd be very happy to. I'll PM you for arrangements.